packages/core/src/components/HelpDrawer/HelpDrawer.scss

```diff
@@ -40,13 +40,15 @@
   // width: 100%;
     // height: 100vh;
     overflow-y: scroll;
+   
 }
 
 .ds-c-help-drawer__content-container {
   // bottom: 0;
   display: flex; // Flex layout helps stick the footer to the bottom
   flex-direction: column;
-  height: calc(100vh - 60px);
+  padding-bottom: 80px;
```

Comment:
I don't think we can use styles that set a specific value because both footer and body can be arbitrary lengths. We'll have to look into something other than flexbox too because of IE11 :/ 

<img width="353" alt="Screen Shot 2020-01-22 at 11 00 48 AM" src="https://user-images.githubusercontent.com/5424713/72915837-90ffc200-3d06-11ea-9aa0-e3cfc2b52824.png">